src/commonTest/kotlin/suite/TestSuiteSearch.kt

```diff
@@ -46,13 +49,33 @@ internal class TestSuiteSearch {
             index.apply {
                 tasks += setSettings(settings)
                 tasks += saveObjects(objects)
-
                 tasks.wait().all { it is TaskStatus.Published }.shouldBeTrue()
-                search(Query("algolia")).nbHits shouldEqual 2
-                val hits = search(Query("elon", clickAnalytics = true)).hits
 
-                hits.shouldNotBeNull()
-                hits.shouldNotBeEmpty()
+                search(Query("algolia")).apply {
+                    nbHits shouldEqual 2
+                    getObjectIDPosition(ObjectID("nicolas-dessaigne")) shouldEqual 0
+                    getObjectIDPosition(ObjectID("julien-lemoine")) shouldEqual 1
+                    getObjectIDPosition(ObjectID("unknown")) shouldEqual -1
```

Comment:
nit: in the CTS, the given string is `""` (empty string) not `"unknown"`

Reply:
My `ObjectID` class throws if I pass an empty string, is the reason why I put `unknown`